tools/dds/dds-adapter/lrs-device-controller.cpp

```diff
@@ -198,26 +216,19 @@ std::vector< std::shared_ptr< realdds::dds_stream_server > > lrs_device_controll
             continue;
         }
 
-        // Set stream metadata support (currently if the device supports metadata all streams does)
-        // Must be done before calling init_profiles()
-        if( _md_enabled )
-        {
-            server->enable_metadata();
-        }
-
-        server->init_profiles( profiles, default_profile_index );
-
         // Set stream intrinsics
```

Comment:
Remove this comment, it is now redundant and does not relate to metadata handling in this `if` statement

Reply:
Removed.